src/components/animated/_liquid.less

```diff
@@ -0,0 +1,93 @@
+.liquid-btn {
+  position: relative;
+  z-index: -1;
+
+  &::before {
+    content: "";
+    width: 400px;
+    height: 400px;
+    position: absolute;
+    top: -80px;
+    left: 50%;
+    transform: translate(-50%, -75%);
+    border-radius: 45%;
+    background: rgba(255, 255, 255, 1);
+    animation: waterEffect 5s linear infinite;
+    z-index: -1;
+  }
+
+  &::after {
+    content: "";
+    width: 400px;
+    height: 400px;
+    position: absolute;
+    top: -80px;
+    left: 50%;
+    transform: translate(-50%, -75%);
+    border-radius: 40%;
+    background: rgba(255, 255, 255, 0.5);
+    animation: waterEffect 10s linear infinite;
+    z-index: -1;
+  }
+
+  .liquidBtn(@color) {
+    border-color: @color;
+    color: #000 !important;
+    .bc(@color);
+
+    &:hover::before {
+      transition: 0.5s linear;
```

Comment:
Can you place transition inside the original implementation of `::before` (not on hover) as it will make the transition cleaner when you move your mouse outside of the button